scripts/read_motor_params.py

```diff
@@ -0,0 +1,38 @@
+params = {
```

Comment:
Same

Reply:
That's why these are in their own directory.